RELEASE.md

```diff
@@ -1,6 +1,14 @@
 # Release History
 *****************
 
+## Release ONDEWO CSI Python Client 2.9.0
+
+### New Features
+* [[OND211-2039]](https://ondewo.atlassian.net/browse/OND211-2039) - Improved automated release process
```

Comment:
I don't know what is usual in CSI, but what is the subject? As far as I know, the release notes should contain notes in the following structure: "ticket" - "subject". "description"